sass/styles/main.scss

```diff
@@ -516,6 +516,14 @@ cq-context {
 	font-family: $base-font;
 	font-size: $font-size-base;
 	overflow: hidden;
+	&.cq-dialog-context{
```

Comment:
@brucebinary You wanted to Dropdowns and Dialogs to  fill the screen on mobile!? nobody said anything about any header!

Reply:
@MahboobehMohammadi I'm not sure you follow. Whatever feature you develop, you need to take to consideration how it will work in binary-static, or even binary-bot. Yes, we do want it to fill the screen, but we only _only_ the dialog to fill the screen, not the chart.

Reply:
@brucebinary  i add the `&.cq-dialog-context` class to the chart only when any dialog is open, when the dialog closes i remove this class, so we shouldnt have any problem.

Reply:
@MahboobehMohammadi I'm aware. But the behaviour is still not what we want. Recall again, and overlays are semi-transparent. We expect when we open a dialog, _just_ the dialog opens, nothing else in the background changes. Would you not expect that as well?